### ISSUE TITLE
Suggesting new child term of "Abnormality of the sacrum"

### ISSUE THREAD

**Issue by theoriste** [obophenotype/human-phenotype-ontology#3685]
Suggest creating new child term of Aplasia/Hypoplasia of the sacrum [HP:0008517] 
New term label: Sacral agenesis
New term definition:  Missing sacrum, sometimes with other missing segments of the  lumbar spine
New term comment (if any):
New term synonyms(if any): absent sacrum; missing sacrum
New term lay person synonyms (if any): absent sacrum
Reference (e.g., PubMed ID): https://books.google.com/books?id=oG5Fn4Ucus4C&pg=PA207&lpg=PA207
Your biocurator ID for nanoattribution (if desired):

**Comment by theoriste** [comment]
Not sure if this should be under aplasia/hypoplasia, or imply abnormalities of the sacrum.

**Comment by pnrobinson** [comment]
I think this term is what you mean
https://hpo.jax.org/app/browse/term/HP:0010305
It currently has the synonym Sacrococcygeal agenesis but I think this is synonymous because aplasia of the sacrum would imply aplasia of the coccyx -- we will add Sacral agenesis as a synonym

**Comment by theoriste** [comment]
Thank you!
